packages/mux-player/src/template.ts

```diff
@@ -26,8 +26,16 @@ const isLive = (props: MuxTemplateProps) => [StreamTypes.LIVE, StreamTypes.LL_LI
 const isLiveOrDVR = (props: MuxTemplateProps) =>
   [StreamTypes.LIVE, StreamTypes.LL_LIVE, StreamTypes.DVR, StreamTypes.LL_DVR].includes(props.streamType as any);
 
-export const content = (props: MuxTemplateProps) => html`
+export const content = (props: MuxTemplateProps) => {
+  const hasSrc = !!props.src
+    ? !!props.src
+    : props.playbackId
+    ? !!getSrcFromPlaybackId(props.playbackId, { domain: props.customDomain, token: props.tokens.playback })
```

Comment:
question(blocking): I think we don't need `getSrcFromPlaybackId` in this nested ternary? Unless I'm missing something, we should be able to just check:

```js
const hasSrc = !!(props.src || props.playbackId);
```

Related: Should this just live in `getProps()`?

Reply:
yeah, maybe we `hasSrc` should move into `getProps()` at this point.